recipes/allegro5/all/test_package/al_ex_color2.c

```diff
@@ -0,0 +1,367 @@
+#define ALLEGRO_UNSTABLE
+#include <allegro5/allegro.h>
```

Comment:
consider reducing the example
it might be enough to just call a single function like `al_init_font_addon` to ensure example compile and links against the library